Entra support documentation for Azure database for PostgreSQL (#126593)
Azure/azure-content: articles/data-factory/connector-azure-database-for-postgresql.md

|:--- |:--- |:--- |
| type | The type property must be set to: **AzurePostgreSql**. | Yes |
| version | The version that you specify. The value is `2.0`. | Yes |

> | authenticationType | Select from basic, service principal authentication, user-assigned managed identity or system-assigned managed identity | Yes |

Reviewer: ```suggestion
| authenticationType | Select from basic,  service principal, system-assigned managed identity and user-assigned managed identity authentication types | Yes |
```

    }
}
``` 

> ### User-assigned managed identity

Reviewer: ```suggestion
### User-assigned managed identity authentication
```

> 

Reviewer: Switch the order for user  User-assigned managed identity authentication & System-assigned managed identity authentication as per the order of the options displayed in the UI.

> A data factory or Synapse workspace can be associated with a [user-assigned managed identities](/azure/data-factory/data-factory-service-identity#user-assigned-managed-identity) that represents the service when authenticating to other resources in Azure. You can use this managed identity for Azure database for PostgreSQL authentication. The designated factory or Synapse workspace can access and copy data from or to your database by using this identity.

Reviewer: ```suggestion
A data factory or Synapse workspace can be associated with a [User-assigned managed identity](/azure/data-factory/data-factory-service-identity#user-assigned-managed-identity) that represents the service when authenticating to other resources in Azure. You can use this managed identity for Azure database for PostgreSQL authentication. The designated factory or Synapse workspace can access and copy data from or to your database by using this identity.
```

    1. Select **Settings** and then **Managed Identities**
    1. Under the **User assigned** tab. Click on the **+ Add** and select your user-managed identity

        :::image type="content" source="media/connector-azure-database-for-postgresql/data-factory-user-identity-configuration.png" alt-text="Screenshot of the user-assigned managed identity configuration in the Azure Data Factory resource." lightbox="media/connector-azure-database-for-postgresql/data-factory-user-identity-configuration.png":::

1. Configure an Azure database for PostgreSQL linked service.

```json
{
    "name": "AzurePostgreSqlLinkedService",
    "type": "Microsoft.DataFactory/factories/linkedservices",
    "properties": {
        "annotations": [],
        "type": "AzurePostgreSql",
        "version": "2.0",
        "typeProperties": {
            "server": "<server name>",
            "port": 5432,
            "database": "<database name>",
            "sslMode": 2,
            "authenticationType": "UserAssignedManagedIdentity",
            "credential": {
                "referenceName": "<your credential>",
                "type": "CredentialReference"
            }
        }
    }
}
```

> ### System-assigned managed identity

Reviewer: ```suggestion
### System-assigned managed identity authentication
```

> 

Reviewer: ```suggestion

A data factory or Synapse workspace can be associated with a [System-assigned managed identity](/azure/data-factory/data-factory-service-identity#system-assigned-managed-identity) that represents the service when authenticating to other resources in Azure. You can use this managed identity for Azure database for PostgreSQL authentication. The designated factory or Synapse workspace can access and copy data from or to your database by using this identity.

```
